lib/ascii_binder/helpers.rb

```diff
@@ -118,8 +123,47 @@ def build_config
       validate_config(YAML.load_stream(open(build_config_file)))
     end
 
+    def create_new_repo
+      gem_template_dir = File.expand_path("../../../templates", __FILE__)
```

Comment:
This is fine, but you might want to just have a constant somewhere very early that defines the REPO_ROOT, upon which you can join the templates directory.  If we ever have to move this file or split it, this could get ugly.